backend/de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/ddl/public/views/M_ProductPrice_CopySource_v.sql

```diff
@@ -1,102 +1,77 @@
-﻿drop view if exists M_ProductPrice_CopySource_v;
```

Comment:
the differs a lot from both intensive_care_uat and master. i copied them both to scratch files and formatted them, but even so i have a hard time spotting the "real" differences.